docs/getting-started-guides/windows/index.md

```diff
@@ -1,139 +1,144 @@
+- Improved support for Pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per Pod
+- Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.5.
+ The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server version 1709.
+
+**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.8.
+
+**Note:** There is one outstanding PR ([51063 Fixes to enable Windows CNI](https://github.com/kubernetes/kubernetes/pull/51063))which has not been merged into v1.8 and is required for Windows CNI to work with kubelet. Users will need to build a private kubelet binary to consume this change. Please refer to these [instructions](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md) for build
 
 ## Prerequisites
-In Kubernetes version 1.5, Windows Server Containers for Kubernetes is supported using the following:
+In Kubernetes version 1.8, Windows Server Containers for Kubernetes is supported using the following:
 
-1. Kubernetes control plane running on existing Linux infrastructure (version 1.5 or later).
+1. Kubernetes control plane running on existing Linux infrastructure (version 1.8 or later).
 2. Kubenet network plugin setup on the Linux nodes.
-3. Windows Server 2016 (RTM version 10.0.14393 or later).
-4. Docker Version 1.12.2-cs2-ws-beta or later for Windows Server nodes (Linux nodes and Kubernetes control plane can run any Kubernetes supported Docker Version).
+3. Windows Server version 1709 (RTM version 10.0.16299.15 or later).
+4. Docker Version 17.06.1-ee-2 or later for Windows Server nodes (Linux nodes and Kubernetes control plane can run any Kubernetes supported Docker Version).
 
 ## Networking
-Network is achieved using L3 routing. Because third-party networking plugins (e.g. flannel, calico, etc) don't natively work on Windows Server, existing technology that is built into the Windows and Linux operating systems is relied on. In this L3 networking approach, a /16 subnet is chosen for the cluster nodes, and a /24 subnet is assigned to each worker node. All pods on a given worker node will be connected to the /24 subnet. This allows pods on the same node to communicate with each other. In order to enable networking between pods running on different nodes, routing features that are built into Windows Server 2016 and Linux are used.
+There are several supported network configurations with Windows Server version 1709 and K8s v1.8 including both Layer-3 routed and overlay topologies using third-party network plugins. 
+1. Upstream L3 Routing - IP routes configured in upstream ToR
```

Comment:
Direct links to each section? :D

Reply:
Added direct links in https://github.com/JMesser81/kubernetes.github.io/pull/1